infrastructure/kube/keep-test/geth-node/eth-goerli-statefulset.yaml

```diff
@@ -0,0 +1,38 @@
+---
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: geth-goerli
+spec:
+  replicas: 1
+  serviceName: geth-goerli
+  volumeClaimTemplates:
+    - metadata:
+        name: geth-goerli
+      spec:
+        accessModes: [ReadWriteOnce]
+        storageClassName: geth-goerli
+        resources:
+          requests:
+            storage: 100Gi
```

Comment:
Let's remember to verify how much disk space the node uses once it's synced and align this setting if necessary.